components/NewsFilter.vue

```diff
@@ -48,7 +48,7 @@ import { countries } from '~/assets/coutries'
   },
 })
 export default class NewsFilter extends Vue {
-  @Prop() reloadNews!: Function
+  @Prop(Function) reloadNews!: Function
```

Comment:
we usually avoid passing function in the prop as we have emit function 
here we can simply trigger the emit event from the child and call the required function on that emit event